capa/features/__init__.py

```diff
@@ -125,6 +125,10 @@ class String(Feature):
     def __init__(self, value, description=None):
         super(String, self).__init__(value, description=description)
 
+    def get_value_str(self):
+        """ """
+        return repr(self.value).strip("'")
```

Comment:
there may be a better way to complete this conversion, I am open to suggestions.

Reply:
hm, this is tricky. seems like a good first past. if someone complains about edge cases, maybe we re-assess then?

there's an edge case for strings that contain single quotes:
![image](https://user-images.githubusercontent.com/156560/110701815-e15a0d80-81ae-11eb-8a40-05d5a496c137.png)

Reply:
instead of `strip` using `repr(s)[1:-1]` appears to hold:

```python
STR = (
    '\t"a"\t',
    "\t'a'\t",
    '\t\'a\'\t',
    'Immanuel Can\'t',
    "C:\\Windows",
    "http://www.example.com/robots.txt",
    "hello\r\nworld"
)

for s in STR:
    print(repr(s)[1:-1])
```

output:

```
\t"a"\t
\t'a'\t
\t'a'\t
Immanuel Can't
C:\\Windows
http://www.example.com/robots.txt
hello\r\nworld
```

we do encounter a side effect that backward slashes are now escaped in the output, e.g Windows paths.

Reply:
this is a lot better, nice job!